src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -141,6 +144,54 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Internal
 
+(def p-key-perms
```

Comment:
Cosmetic - might be good to just call these `private-key...` instead of `p-key...`.